packages/rum-recorder/src/domain/rrweb/observer.ts

```diff
@@ -537,7 +526,7 @@ export function initObservers(o: ObserverParam, hooks: HooksParam = {}): Listene
   const fontObserver = o.collectFonts ? initFontObserver(o.fontCb) : noop
 
   return () => {
-    mutationObserver.disconnect()
+    mutationBuffer.stop()
```

Comment:
what about returning a handler to be consistent with the others?

Reply:
Sure